src/base/projects/PeerSelector.svelte

```diff
@@ -11,15 +11,20 @@
 
   let meta: Peer | undefined;
   // List of items to be created for the Dropdown component.
-  let items: { key: string; value: string; badge: string | null }[] = [];
+  let items: { key: string; value: string; title: string; badge: string | null }[] = [];
+
+  function createTitle(p: Peer): string {
+    const name = p.person?.name ? p.person.name : p.id;
+    return p.delegate ? `${name} is the delegate peer of this project` : `${name} is a tracked peer by the delegate`;
```

Comment:
```suggestion
    return p.delegate ? `${name} is a delegate of this project` : `${name} is a peer tracked by this seed`;
```

Reply:
Done in [827163c](https://github.com/radicle-dev/radicle-interface/pull/331/commits/827163c6a93d470825a3d8973c3dc5c3f0d38e82)